File: POO UML/Codigos/Aula 1/c-sharp/project-conta01/AbrirConta.cs
using System;

namespace project_conta01
{
    internal class AbrirConta
    {
        internal void Abrir()
        {

            // Instânciando a classe contaCorrente
            ContaPoupanca cliente1 = new ContaPoupanca();
            ContaPoupanca cliente2 = new ContaPoupanca();

            // Criando uma conta corrente
            cliente1.Agencia = 1234;
            cliente1.NumeroConta = "123456";
            cliente1.DiaDeposito = 10;
            cliente1.Depositar(50);

            cliente2.Agencia = 1234;
            cliente2.NumeroConta = "123456";
            cliente2.DiaDeposito = 10;
            cliente2.Depositar(50);
            
            ExibirDadosBancarios(cliente1);
            ExibirDadosBancarios(cliente2);
        }

        private void ExibirDadosBancarios(ContaPoupanca cliente)
        {
            Console.WriteLine("Qual é o dia de deposito? ");
            Console.WriteLine(cliente.DiaDeposito);
            Console.WriteLine("Qual é o Numero da Conta? ");
            Console.WriteLine(cliente.NumeroConta);
            Console.WriteLine("Qual é o Agencia? ");
            Console.WriteLine(cliente.Agencia);
            Console.WriteLine("Meu Saldo? ");
            Console.WriteLine(cliente.Saldo(1234));
        }
        }

        
}

File: Outros/POO UML/Codigos/Aula 1/Exemplo banco/java/Conta/src/AbrirConta.java
public class AbrirConta {
    public void Abrir()
    {
        //Instânciação da classe ContaCorrente
        ContaPoupanca cliente1 = new ContaPoupanca();
        ContaPoupanca cliente2 = new ContaPoupanca();

        // Criando uma conta corrente
        cliente1.numeroConta = "123";
        cliente1.agencia = 1;
        cliente1.DiaDeposito = 10;

        cliente2.numeroConta = "999";
        cliente2.agencia = 2;
        cliente2.DiaDeposito = 5;

        System.out.println("Agência: " + cliente1.agencia);   
        System.out.println("Numero Conta: " + cliente1.numeroConta);   
        System.out.println("Dia deposito: " + cliente1.DiaDeposito);   
        System.out.println("Agência: " + cliente2.agencia);   
        System.out.println("Numero Conta: " + cliente2.numeroConta);   
        System.out.println("Dia deposito: " + cliente2.DiaDeposito);   
    }
}
